celery/worker/job.py

```diff
@@ -437,8 +437,14 @@ def on_failure(self, exc_info):
                     self._announce_revoked(
                         'terminated', True, string(exc), False)
                     send_failed_event = False  # already sent revoked event
+
             # (acks_late) acknowledge after result stored.
-            if self.task.acks_late:
+            if self.task.acks_late and isinstance(exc, WorkerLostError):
+                #: For acks_late set to True, we changed the default behavior
+                #: to handle worker crash. Currently, we allow the message to be rejected
+                #: and requeued so it will be executed again by another worker.
+                self.reject(requeue=True)
```

Comment:
I think the original idea was not to requeue it, but to just nack it, @grappeq can you confirm? 
Aga, have you checked that if the task is not requeued but just nacked then the import does not get stuck?

Reply:
I checked only the behaviour with reject, in which a  import has a chance to finish successfully. In all my cases that I have tested, crashed import finished successfully when I was rejecting them and requeuing.

Reply:
I think we should requeue, not nack. Afaik nacking means that the task will never be executed which e.g. for user imports will cause the import to get stuck. On the other hand we are fine with requeuing as `acks_late` tasks either way have to be idempotent so it being executed twice is an expected scenario.

Reply:
Makes sense, I think the code in other request.py would actually nack (unless we set some flag), thus my question. But I agree with requeuing